### ISSUE TITLE
Any thoughts on using a static site generator?

### ISSUE THREAD

**Issue by nayaabkhan** [airyxos/website#3]
We're using a pretty basic setup which is totally OK to begin. But possibly we should use some [static site generator](https://jamstack.org/generators/)? It would scale well.

I can suggest a few from past experience:

- [**11ty**](https://www.11ty.dev): [Simple](https://www.11ty.dev/#quick-start), [extensible](https://www.11ty.dev/docs/plugins/), [offers several templating choices](https://www.11ty.dev/docs/languages/), [data files](https://www.11ty.dev/docs/data-js/), [RSS support](https://www.11ty.dev/docs/plugins/rss/), [Shortcodes](https://www.11ty.dev/docs/shortcodes/)
  Pretty feature-rich, configurable, and not opinionated. Suitable for almost all kinds of sites.
- [**Zola**](https://www.getzola.org/): [Fast](https://www.rust-lang.org/), [Shortcodes](https://www.getzola.org/documentation/content/shortcodes/), [I18n](https://www.getzola.org/documentation/content/multilingual/), [RSS](https://www.getzola.org/documentation/templates/feeds/), [Opinionated structure](https://www.getzola.org/documentation/getting-started/directory-structure/)
  Opinionated, fewer features, and opinionated. More suitable for a blog, in my opinion.

**Comment by llui85** [comment]
Another option could be [SvelteKit](https://kit.svelte.dev), which compiles to a static site (but still works without JS).

**Comment by mszoek** [comment]
Yep - a static site generator would be great. Whatever you think is appropriate is fine with me. It just needs to run on FreeBSD (our build machine) and deploy only HTML/CSS and client-side JS. I'm using surge.sh for hosting so nothing server-side.
